addons/mail/static/src/scss/shared_classes.scss

```diff
@@ -0,0 +1,11 @@
+// Regrup CSS classes shared across several components.
+// Thes rules can be used by modules with mail dependecy.
+
+
+// == Thread avatar image
+//------------------------------------------------------------------------------
+.o_avatar_thread {
```

Comment:
I don't like the name and the need of this class. It's oddly too specific to be a generic one, and currently you only use it in the livechat legacy file, but the goal is to remove that file completely to re-use existing components instead.

I would probably not change anything here for now, as you are just making a move from an obsolete file to a generic place that we probably don't need.

Reply:
> I don't like the name and the need of this class. 

Well, it's the name that you thought didn't match `web`, but was ok for `mail` (https://github.com/odoo/odoo/pull/82590#discussion_r788976302).
What name do you prefer? 🙂

> and currently you only use it in the livechat legacy file [...]  you are just making a move from an obsolete file to a generic place that we probably don't need.

Nope, we already have 2 occurrences;  livechat and `o_Activity ` (https://github.com/odoo/odoo/pull/82590).
Soon it will be used also in `o_Message_authorAvatar`, `o_Composer_currentPartnerAvatar` etc..

Reply:
But do we want all other avatars that you listed to be this exact same size? Because it's pretty much the only thing contained in this class.

If this is the basic size we want for all avatars, then I would go with `o_avatar` as suggested above, if this is a specific version, maybe `o_avatar_${some size adjective to be determined}`?